files/common/var/lib/delphix-platform/os-migration

```diff
@@ -47,6 +47,13 @@ function perform_migration() {
 	chown -R root:root /var/delphix/server ||
 		die "Failed to chown /var/delphix/server"
 
+	if [[ -d /var/delphix/dmsuite ]]; then
+		mv /var/delphix/dmsuite /var/delphix/masking ||
```

Comment:
Idk if this matters, but I think this will put “dmsuite” under “/var/delphix/masking”, if the “masking” directory already exists...

Reply:
Hmm, that's a good point. We should probably check for that.

Reply:
Good point. It shouldn't matter, but I'll change it to be `mv -T` to be safe.